src/main/include/subsystems/Intake.hpp

```diff
@@ -91,6 +91,10 @@ class Intake : public SubsystemBase {
     void RobotPeriodic() override;
 
 private:
+    // Arm solenoid Positions
+    static constexpr int kArmForward = 0;
+    static constexpr int kArmReverse = 1;
```

Comment:
These are redundant since the member function itself documents what true and false correspond to.